src/Form/Question.php

```diff
@@ -154,7 +154,7 @@ private function prepareInput(&$input)
 
         if ($question_type) {
             if (isset($input['default_value'])) {
-                $input['default_value'] = $question_type::formatDefaultValueForDB($input['default_value']);
+                $input['default_value'] = (new $question_type())->formatDefaultValueForDB($input['default_value']);
```

Comment:
`$question_type` is already an instance of `QuestionTypeInterface`.
```suggestion
                $input['default_value'] = $question_type->formatDefaultValueForDB($input['default_value']);
```